listenbrainz/webserver/views/dump_api.py

```diff
@@ -0,0 +1,62 @@
+from flask import Blueprint, request, jsonify
+from listenbrainz.webserver.errors import APIBadRequest, APINotFound
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+import listenbrainz.db.dump as db_dump
+
+dump_api_bp = Blueprint("dump_api_v1", __name__)
```

Comment:
Hmm, I'm not a fan of that name. I think we should think bigger and make a more generic "health-stats" or some such thing that doesn't narrow the scope too much.

Reply:
Done after IRC discussion!